pkg/ingester/limiter.go

```diff
@@ -92,10 +82,15 @@ func (l *Limiter) maxMetadataPerUser(userID string) int {
 	return l.convertGlobalToLocalLimitOrUnlimited(userID, l.getShardSize(userID), l.limits.MaxGlobalMetricsWithMetadataPerUser)
 }
 
+func (l *Limiter) maxExemplarsPerUser(userID string) int {
+	// We don't use `convertGlobalToLocalLimitOrUnlimited`, because we don't want "unlimited" part. 0 means disabled.
+	return l.ringStrategy.convertGlobalToLocalLimit(l.getShardSize(userID), l.limits.MaxGlobalExemplarsPerUser(userID))
```

Comment:
If we're unable to convert the global limit to local limit (e.g. because the ingesters ring client hasn't updated yet and see no ingesters yet) then we return 0, which means exemplars are disabled.